src/app/public/modules/href/types/app-route.ts

```diff
@@ -0,0 +1,7 @@
+export interface AppRoute {
```

Comment:
Since this is part of the public API, it needs a `Sky` prefix. Maybe it could just be called `SkyHref` since it's the result of calling `resolveHref`?